components/bl-autocomplete-component/src/options/index.js

```diff
@@ -1,29 +1,24 @@
-import { useRef, useState } from 'react';
+import { useState, useRef } from 'react';
 
 import { Option } from './option';
 
-import { generateId } from '../helpers';
 import { useOptionsPlacement } from '../helpers';
 
 const { cn } = BackendlessUI.CSSUtils;
 
-export const Options = props => {
-  const {
-    optionList,
-    autocompleteHeight,
-    setInputValue,
-    setAutocompleteValue,
-    setIsOptionsOpen,
-    onAutocompleteChange,
+export function Options(props) {
+  const { optionList, autocompleteHeight,
+    setInputValue, setAutocompleteValue, setIsOptionsOpen, onAutocompleteChange
   } = props;
-  const optionsRef = useRef();
+
+  const optionsRef = useRef(null);
```

Comment:
That looks more like optionsContainerRef or something like it, but not an options